builtin/v10/miner/miner_types.go

```diff
@@ -369,3 +369,35 @@ type SectorClaim struct {
 	MaintainClaims []verifreg.ClaimId
 	DropClaims     []verifreg.ClaimId
 }
+
+type GetOwnerReturn struct {
+	Owner    addr.Address
+	Proposed *addr.Address
+}
+
+type IsControllingAddressParams struct {
+	Address addr.Address
+}
+
+type IsControllingAddressReturn struct {
+	IsControlling bool
+}
+
+type GetSectorSizeReturn struct {
+	SectorSize abi.SectorSize
+}
+
+type GetAvailableBalanceReturn struct {
+	AvailableBalance abi.TokenAmount
+}
+
+// TODO check this works
```

Comment:
I think the best way to do this is to just generate a serialization of some VestingFunds from built-in actors, and dump that hard-coded into a test here.